package.json

```diff
@@ -6,9 +6,10 @@
   "version": "10.2.0-pre",
   "repository": "github:grafana/grafana",
   "scripts": {
-    "build": "yarn i18n:compile && NODE_ENV=production webpack --progress --config scripts/webpack/webpack.prod.js",
+    "prebuild": "yarn i18n:compile && yarn plugin:build",
```

Comment:
In my environment,  `yarn plugin:build` takes ~13-15s to build each plugin so it's an important delay when running any command.

Reply:
Don't think we need to run `plugin:build` sequentially to `yarn build`. The artifacts from plugins aren't depended on by core. `yarn plugin:build & yarn build` would likely reduce time taken as they'd be concurrent. 

There are a few other things we can do to speed up the builds but should probably tackle them in another PR.

- Only include the `ForkTsCheckerWebpackPlugin` and `ESLintPlugin` when building / watching for development
- Replace the `ts-loader` in input-datasource webpack config with `swc-loader`
- Set stats to `errors-only` in production else `normal`

Reply:
:+1: will do the first part

Reply:
I actually omitted the intput-datasource plugin too